### ISSUE TITLE
research description disappearance

### ISSUE THREAD

**Issue by c21adano** [HGustavs/LenaSYS#12980]
when you pull out the pop up options for ER entity or relation the description which show keybind disappears and wont appear again until the site is refreshed.

You get the pop up options by holding left click on ER entity or relation. This will show uml stuff.

**Comment by b21sarbo** [comment]
Interpretation/translation of the issue:

- If you pull out the pop up options for an ER entity, the description showing the keybindings disappear.
- If you pull out a relation, the description showing the keybindings disappear.
The description appears when the site is loaded in and when it is refreshed.

- If you hold left mousebutton on an ER entity, the pop up options will appear.
- If you hold left mousebutton on a relation, the pop up options will appear.
The pop up options shows UML stuff.

Conclusion: The issue seem to consist of multiple issues and scenarios. Further inspection of the different scenarios may be needed.

**Comment by b21sarbo** [comment]
I haven’t been able to replicate the scenarios I interpreted from the issue description but I have tested the interpreted scenarios when opening the “diagram.php” file in google chrome.

These are the results: 
- Holding left mousebutton on an ER entity does not make the pop up options appear. 
- Holding left mousebutton on a relation does not make the pop up options appear.
- The description does not appear when the site is loaded in or when it is refreshed.

- Double clicking on an ER entity will reveal an options menu for ER entities on the right side of the website.
- Double clicking on a relation will reveal an options menu for relations on the right side of the website.
- The descriptions including the keybindings are revealed by hovering over one of the tools in the options menu or in the modes menu on the left side of the website.
